### ISSUE TITLE
Episode 9 , ffmpeg return error

### ISSUE THREAD

**Issue by maxya** [gyoridavid/ai_agents_az#7]
using container gyoridavid/narrated-story-creator:latest-cuda with workflow from episode 9

narrator-story  | Using CUDA
narrator-story  | INFO:     Started server process [1]
narrator-story  | INFO:     Waiting for application startup.
narrator-story  | INFO:     Application startup complete.
narrator-story  | INFO:     Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)
narrator-story  | Saved 1 videos to persistent storage
narrator-story  | INFO:     192.168.11.5:44894 - "POST /api/videos HTTP/1.1" 200 OK
narrator-story  | Saved 1 videos to persistent storage
narrator-story  | Downloading background video for 8951b0fe-7283-4259-ac27-b17620f63a96
narrator-story  | Downloading person image for 8951b0fe-7283-4259-ac27-b17620f63a96
narrator-story  | creating overlay
narrator-story  | Text dimensions: 373x52, Y offset: 24
narrator-story  | Overlay saved to /app/tmp/8951b0fe-7283-4259-ac27-b17620f63a96/overlay.png
narrator-story  | creating narration
narrator-story  | WARNING: Defaulting repo_id to hexgrad/Kokoro-82M. Pass repo_id='hexgrad/Kokoro-82M' to suppress this warning.
narrator-story  | /usr/local/lib/python3.10/dist-packages/torch/nn/modules/rnn.py:123: UserWarning: dropout option adds dropout after all but last recurrent layer, so non-zero dropout expects num_layers greater than 1, but got dropout=0.2 and num_layers=1
narrator-story  |   warnings.warn(
narrator-story  | /usr/local/lib/python3.10/dist-packages/torch/nn/utils/weight_norm.py:143: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
narrator-story  |   WeightNorm.apply(module, name, dim)
narrator-story  | Collecting en-core-web-sm==3.8.0
narrator-story  |   Downloading https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.8.0/en_core_web_sm-3.8.0-py3-none-any.whl (12.8 MB)
narrator-story  |      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 12.8/12.8 MB 17.4 MB/s eta 0:00:00
narrator-story  | Installing collected packages: en-core-web-sm
narrator-story  | Successfully installed en-core-web-sm-3.8.0
narrator-story  | WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
narrator-story  | ✔ Download and installation successful
narrator-story  | You can now load the package via spacy.load('en_core_web_sm')
narrator-story  | creating subtitle
narrator-story  | Subtitle file created: /app/tmp/8951b0fe-7283-4259-ac27-b17620f63a96/subtitle.srt
narrator-story  | rendering video
narrator-story  | ffmpeg: Error while filtering: Operation not permitted
narrator-story  | ffmpeg: frame=    0 fps=0.0 q=0.0 Lsize=       0kB time=N/A bitrate=N/A speed=N/A
narrator-story  | ffmpeg: Conversion failed!

**Comment by maxya** [comment]
Not super sure about how to include all post data the best way, here's a link to exported workflow with pinned nodes (with data) if this helps. 
[Download workflow](https://drive.google.com/file/d/1SnsGc7W3yP8k85LxtUdmqT1C1V0JTNP6/view?usp=sharing )

Since you merged most of this to[ this repo](https://github.com/gyoridavid/ai-agents-no-code-tools) , I'll probably going to be migrating workflow to use it then.
